person.rb

```diff
@@ -10,8 +10,9 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
-  attr_reader :id
+  attr_reader :id, :rentals
   attr_accessor :name, :age
 
```

Comment:
- [ ] Kindly add a method for adding the book being rented to rentals. the person itself should be sent as a parameter to create the new rental achieving the 'has-many' association